doc/apiref.md

```diff
@@ -97,7 +97,7 @@ definition.
     site_pp = create_site_pp(master, :node_def_name => 'puppet_agent', :manifest => 'notify { hello: }')
     ```
 
-### `set_perms_on_remote`
+### `set_perms_on_remote(host, path, mode)`
```

Comment:
The options hash is only documented in the examples. It would be nice to have a full description of what options are available.